### PR TITLE
Update pat.md to add configuration instructions

### PR DESCRIPTION
**Pull request by beirtipol**
Add instructions about placing the token in the config file! These instructions are available on other pages but it makes sense to include them in the step-by-step guide?

### CODE REVIEW

**Review by ludeeus — CHANGES_REQUESTED** [review]
This will add duplicate information in two different places that needs to be kept in sync, options:
- just reference the main config
- link to main config

**Comment by beirtipol** [comment]
Which page should it link to? I'm struggling finding the page again that I originally saw it on. Seems like the 'pat' information should all be in one place as the only time you would edit the token in configuration.yml is after you generate a new one?

**Comment by ludeeus** [comment]
https://hacs.xyz/docs/configuration/legacy

File: documentation/configuration/pat.md [hunk 1]
```diff
@@ -48,9 +48,4 @@ Now you see the generated token, this will be the **only** time you see it, make
 
 ## Step 6 - Edit configuration.yml
 
-Edit /config/configuration.yml and add the following section:
-
-```
-hacs:
-  token: the_token_you_just_copied
-```
+Follow the instructions [here](legacy.md) to add the token to the Hassio configuration.yml
```
Comment (ludeeus) [inline documentation/configuration/pat.md:51]:
```suggestion
Follow the [instructions here](legacy.md) to add the token to the Home Assistant configuration
```

"hassio" has not existed in almost 2 years

**Review by ludeeus — APPROVED** [review]

**Pull request merged**